From ef6a147ea381e76fa02358aa5b65816121d57a4c Mon Sep 17 00:00:00 2001
From: Taehee Yoo <ap420073@gmail.com>
Date: Sun, 21 Aug 2016 20:38:22 +0900
Subject: [PATCH] rtl8xxxu: Fix rtl8188eu connection fail

rtl8188eu vendor driver's LLT init routine is similar
rtl8xxxu_init_llt_table() than rtl8xxxu_auto_llt_table().

So now, rtl8188eu can connect to AP.

Signed-off-by: Taehee Yoo <ap420073@gmail.com>
Signed-off-by: Jes Sorensen <Jes.Sorensen@redhat.com>
---
 drivers/net/wireless/realtek/rtl8xxxu/rtl8xxxu_8188e.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/wireless/realtek/rtl8xxxu/rtl8xxxu_8188e.c
+++ b/drivers/net/wireless/realtek/rtl8xxxu/rtl8xxxu_8188e.c
@@ -1230,7 +1230,7 @@ struct rtl8xxxu_fileops rtl8188eu_fops =
 	.power_on = rtl8188eu_power_on,
 	.power_off = rtl8xxxu_power_off,
 	.reset_8051 = rtl8188eu_reset_8051,
-	.llt_init = rtl8xxxu_auto_llt_table,
+	.llt_init = rtl8xxxu_init_llt_table,
 	.init_phy_bb = rtl8188eu_init_phy_bb,
 	.init_phy_rf = rtl8188eu_init_phy_rf,
 	.phy_iq_calibrate = rtl8188eu_phy_iq_calibrate,
